### PR TITLE
Module 1 PR

### PR DESCRIPTION
**Pull request by fran-man**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,12 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from session_items import get_items, get_item, add_item, save_item
```
Comment (Jonesey13) [inline app.py]: I'd probably keep this as session given that we've got some similar app method names floating around here

File: app.py [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return renderIndex()
```
Comment (Jonesey13) [inline app.py]: Why not just move `render_index()` into this method?

Remember that you can use `redirect("/")`  for the other routes.

File: app.py [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+
+@app.route('/listitem', methods=['POST'])
+def addListItem():
+    print("Adding Item!")
+    print('Title=' + request.form.get('textbox'))
+    add_item(request.form.get('textbox'))
+    return renderIndex()
```
Comment (Jonesey13) [inline app.py]: You'll want to redirect to keep the searchbar url consistent

File: app.py [hunk 4]
```diff
@@ -1,12 +1,33 @@
...
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return renderIndex()
+
+@app.route('/listitem', methods=['POST'])
```
Comment (Jonesey13) [inline app.py]: Shouldn't this route be `/add-item` or something similar?

File: templates/index.html [hunk 5]
```diff
@@ -11,8 +11,18 @@ <h1 class="display-4">To-Do App</h1>
...
+          {% for item in list %}
+              <li>{{ item.title }} - {{ item.status }}<form action="/completeditem" method="post">
+                  <input type="hidden" name="id" name="id" value="{{ item.id }}">
+                  <button type="submit">Done!</button>
+              </form>
+              </li>
+          {% endfor %}
       </ul>
+    <form action="/listitem" method="post">
+        <input type="text" id="textbox" name="textbox">
```
Comment (Jonesey13) [inline templates/index.html:23]: I'd add a `required` attribute here to prevent adding empty todos

**Review by Jonesey13 — APPROVED** [review]
This all looks to be working for me!

There's a couple of suggestions regarding urls and page navigation but feel free to merge this PR in whenever you are ready 👍

File: app.py (hunk 3 above)
Comment (fran-man) [reply app.py]: aha thanks

**Pull request merged**